selfdrive/ui/translations/main_pt.ts

```diff
@@ -0,0 +1,1376 @@
+      &lt;/ol&gt;
+    </translation>
+    </message>
+</context>
+<context>
+    <name>PrimeAdWidget</name>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="187"/>
+        <source>Upgrade Now</source>
+        <translation>Atualizar Agora</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="192"/>
+        <source>Become a comma prime member at connect.comma.ai</source>
+        <translation>Torne-se um membro comma prime em connect.comma.ai</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="199"/>
+        <source>PRIME FEATURES:</source>
+        <translation>PRIME FEATURES:</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>Remote access</source>
+        <translation>Acesso remoto</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>1 year of storage</source>
+        <translation>1 ano de armazenamento</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>Developer perks</source>
+        <translation>Benefícios para desenvolvedor</translation>
+    </message>
+</context>
+<context>
+    <name>PrimeUserWidget</name>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="126"/>
+        <source>✓ SUBSCRIBED</source>
+        <translation>✓ INSCRITO</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="132"/>
+        <source>comma prime</source>
+        <translation>comma prime</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="138"/>
+        <source>CONNECT.COMMA.AI</source>
+        <translation>CONNECT.COMMA.AI</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="151"/>
+        <source>COMMA POINTS</source>
+        <translation>PONTOS COMMA</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../qt/text.cc" line="36"/>
+        <source>Reboot</source>
+        <translation>Reiniciar</translation>
+    </message>
+    <message>
+        <location filename="../qt/text.cc" line="41"/>
+        <source>Exit</source>
+        <translation>Sair</translation>
+    </message>
+    <message>
+        <location filename="../qt/util.cc" line="21"/>
+        <source>dashcam</source>
+        <translation>dashcam</translation>
+    </message>
+    <message>
+        <location filename="../qt/util.cc" line="21"/>
+        <source>openpilot</source>
+        <translation>openpilot</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/util.cc" line="82"/>
+        <source>%n minute(s) ago</source>
```

Comment:
Do you mind adding the plural forms for these?

Reply:
Done!